Errors building examples (specifically notebook and terminal)

From gitter on Friday:

I've played around with the examples a bit more and I think they are making more sense. Will take a swipe at a doc with a few screenshots for the examples. One issue: console, filebrowser, and lab examples work fine. However, notebook and terminal are throwing 404 errors when trying to GET /build/bundle.js

Tried `npm run clean:examples` @jasongrout's suggestion. Still issue with notebook and terminal running after `npm run build:examples`. I'm working with a more recent commit than Jason (mine: commit 06a042a81d1407f484a89fa48f6eaa7187d1f5fe)

FWIW I tried deleting the `node_modules` directory in the notebook example and then `npm run build:examples`. No happiness :-(

Let me know if you want me to retest anything.

The hard refresh didn't help on Chrome and Safari.

I did a new clone in an experiments directory on my system. Receiving this when running ``npm install && npm run build``:
```
> jupyterlab-example-notebook@ build /Users/carol/experiments/jupyterlab/examples/notebook
> tsc --project src && webpack --config webpack.conf.js

src/index.ts(109,56): error TS2345: Argument of type 'MimeData' is not assignable to parameter of type 'MimeData'.
  Types have separate declarations of a private property '_types'.

npm ERR! Darwin 15.5.0
npm ERR! argv "/Users/carol/anaconda/bin/node" "/Users/carol/anaconda/bin/npm" "run" "build"
npm ERR! node v4.2.6
npm ERR! npm  v2.14.12
npm ERR! code ELIFECYCLE
npm ERR! jupyterlab-example-notebook@ build: `tsc --project src && webpack --config webpack.conf.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jupyterlab-example-notebook@ build script 'tsc --project src && webpack --config webpack.conf.js'.
npm ERR! This is most likely a problem with the jupyterlab-example-notebook package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     tsc --project src && webpack --config webpack.conf.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls jupyterlab-example-notebook
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/carol/experiments/jupyterlab/examples/notebook/npm-debug.log
```

--- a/examples/notebook/package.json
+++ b/examples/notebook/package.json
@@ -12,6 +12,7 @@
     "jupyter-js-services": "^0.10.4",
     "jupyterlab": "file:../..",
     "phosphor-commandpalette": "^0.2.0",
+    "phosphor-dragdrop": "^0.9.1",
     "phosphor-keymap": "^0.8.0",
     "phosphor-splitpanel": "^1.0.0-rc.1",
     "phosphor-widget": "^1.0.0-rc.1"

